server/migration/src/m20220811_190845_create_expenses_table.rs

```diff
@@ -101,7 +101,7 @@ impl MigrationTrait for Migration {
             name: Set("Netflix for my little family".to_string()),
             description: Set("Cheapest monthly plan of Netflix - Maybe upgrade later".to_string()),
             value: Set(Decimal::new(2490, 2)),
-            start_date: Set(NaiveDate::from_ymd(2022, 9, 24)),
+            start_date: Set(NaiveDate::from_ymd_opt(2022, 9, 24).unwrap()),
```

Comment:
Is it a good thing? Panicing in a migration make more sense to me, then silently hiding the error.
(Applies to all new from_ymd_opt)

Reply:
Unwrap panic, just without a message, unlike expect. There is no way to hide errors in rust so easily and luckily.

Reply:
Right, I understand now.